LFS171x/docs/the-promise-of-business-blockchain-technologies.md

```diff
@@ -1 +1,347 @@
+
+A blockchain may record a hash of a document. As an example, photographers could place a hash of their unique digital photographs on the blockchain. The hash of a digital photograph will be constant so long as the photograph file has not been altered. Therefore, the blockchain can control and track the distribution of the photograph, detect the introduction of counterfeit images, and be used to resolve disputes as to who first introduced the image. By placing a hash of intellectual property documents on the blockchain, a party can publicly demonstrate data ownership, and prove the existence of certain documents at a given moment in time, without revealing the actual data. In addition to the hash, you may also choose to store the location of the file in the blockchain, which could be used for retrieval.
+
+### Business Interest in Hyperledger (Brian Behlendorf)
+
+[![Business Interest in Hyperledger (Brian Behlendorf)](../images/video-image.png)](https://youtu.be/cDZg8R5z1oQ)
+
+### Finance (Part I)
+
+<p align="center">
+  <i>"Blockchain has huge potential to move the financial services industry away from messaged based models, slow reconciliation processes and inefficiency of fragmented data stores. With blockchain, financial services can move to a shared data construct, driving down costs, increasing efficiency and opening up entirely new business models."</i>
+</p>
+
+<p align="center">
+  <i>- David Treat, Accenture</i>
+</p>
+
+The Bitcoin blockchain was created as a "peer-to-peer electronic cash system" ([Satoshi Nakamoto, Bitcoin](https://bitcoin.org/bitcoin.pdf)). Therefore, the first blockchain use case in existence was payments. However, Bitcoin proved to be quite slow to process payments, "somewhere in the region of 7 transactions per second" ([Guy Brandon, due.com](https://due.com/blog/can-the-blockchain-scale/), February 2017), when compared to Visa, which "averages around 2,000 transactions per second, with peak capacity of perhaps 50,000 transactions per second" ([Guy Brandon, due.com](https://due.com/blog/can-the-blockchain-scale/), February 2017). Developers are actively working to increase the throughput capacity of Bitcoin and other blockchain payment systems ([lightning.network](https://lightning.network/)). Payments, especially international payments, can be quite costly. Blockchain technologies plan to decrease the costs associated with payments, by allowing parties to interact directly, instead of transferring through an intermediary, such as a bank. In addition, having a record of all past payments is useful to auditors and regulators. Financial institutions have heavily researched blockchain payment systems because a universally recorded world state of payment information can decrease the number of payment disputes among institutions. 

```

Comment:
add italics `_italics_` to "peer-to-peer electronic cash system", "somewhere in the region of 7 transactions per second", and "averages around 2,000 transactions per second, with peak capacity of perhaps 50,000 transactions per second"

As in:
```
"_peer-to-peer electronic cash system_"
"_somewhere in the region of 7 transactions per second_"
"_averages around 2,000 transactions per second, with peak capacity of perhaps 50,000 transactions per second_"
```